.vscode/extensions.json

```
@@ -0,0 +1,8 @@
+{
+	"recommendations": [
+        "esbenp.prettier-vscode",
+        "mhutchie.git-graph",
+        "ms-vscode.vscode-typescript-tslint-plugin",   
```

We won't use TSLint only ESLint with configuration for TypeScript, because TSLint is consider depracated since 2019, see https://palantir.github.io/tslint/ :wink:
```suggestion
        "dbaeumer.vscode-eslint",   
```

---

Ok :+1: